src/css/main.less

```diff
@@ -966,24 +967,33 @@ fieldset[disabled] .btn-info.active {
       padding: @base-spacing-unit*2;
       width: @sidebar-width;
 
-      .btn-success {
+      .btn-success, .dropdown-toggle, .dropdown-menu {
```

Comment:
The way I see it is that the sidebar "overrides" the defaults for these guys, so I think it's more readable than introducing a new class